Sources/AppStoreConnectCLI/Services/Operations/ReadCertificateOperation.swift

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ReadCertificateOperation: APIOperation {
+
+    struct Options {
+        let serial: String
+    }
+
+    enum ReadCertificateError: LocalizedError {
```

Comment:
If you're nesting a type like this then you don't need a type like `ReadCertificateError` it could just be `Error`.

Reply:
This is because its actual type name is `ReadCertificateOperation.ReadCertificateError` which seems a bit repetitive when it could be `ReadCertificateOperation.Error`.

Reply:
Admittedly this will introduce a disambiguation issue with `Swift.Error`. You would have to make sure the methods below like `execute ` disambiguated `Error`.

Reply:
Luckily we are slightly protected because the protocol defines `Swift.Error` so you are forced to disambiguate